metricbeat/module/kubernetes/kubernetes.yml

```diff
@@ -43,7 +43,7 @@ spec:
     spec:
       containers:
       - name: kube-state-metrics
-        image: registry.k8s.io/kube-state-metrics/kube-state-metrics:v2.6.0
+        image: registry.k8s.io/kube-state-metrics/kube-state-metrics:v2.7.0
```

Comment:
it also would need test files updates for `state-*` metricsets, maybe move this in another PR?

Reply:
This is the tests I was referring as well Constanca here eg.https://github.com/elastic/beats/tree/main/metricbeat/module/kubernetes/state_container/_meta/test

I agree this can be done on a new PR

Reply:
we should update the template issue for updating k8s version and add this step as well there